src/tests/components/ForecastDetails.test.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import ForecastDetails from "../../components/ForecastDetails";
+
+describe("ForecastDetails", () => {
+  const validProps = {
+    date: 1111111,
+    temperature: {
+      min: 12,
+      max: 22,
+    },
+    humidity: 54,
+    wind: {
+      speed: 5,
+      direction: "w",
+    },
+  };
+  it("renders correctly", () => {
+    const { asFragment } = render(<ForecastDetails forecast={validProps} />);
+    expect(asFragment()).toMatchSnapshot();
+  });
+  it("renders correct values for props", () => {
+    const { getByText } = render(<ForecastDetails forecast={validProps} />);
+    expect(getByText("Thu Jan 01 1970")).toHaveClass("forecast-details__date");
+    expect(getByText("12°C")).toHaveClass("forecast-details__min-temperature");
+    expect(getByText("22°C")).toHaveClass("forecast-details__max-temperature");
+    expect(getByText("54%")).toHaveClass("forecast-details__humidity");
+    expect(getByText("5mph, w")).toHaveClass("forecast-details__wind");
```

Comment:
This queries will not work as they are, as they are looking for the exactly what you pass them to be in an element. For example you are searching for `12°C`, but whats actually rendered to the test DOM is `Min: 12 °C` so the query wont find it.
Swapping this out highlights another issue, which is what you are matching it with. The text is rendered inside a `p` tag, which has no class attached and so the assertion will fail. 
Generally Id say its better to assert whether what you are expecting is actually in the document and not look for classnames as they dont affect the user directly, so you could use the `toBeInTheDocument` matcher instead. This would fix the other assertions here too:
```
    expect(getByText("Min: 12 °C")).toBeInTheDocument();
    expect(getByText("Max: 22°C")).toBeInTheDocument();
    expect(getByText("Humidity: 54%")).toBeInTheDocument();
    expect(getByText("Wind: 5mph, w")).toBeInTheDocument();
```
Note that there is inconsistent spacing needed to get these to work, it would be good to look at the ForecastDetails file and check there arent any spaces where there shouldnt be